src/UCTSearch.h

```diff
@@ -85,10 +88,14 @@ class UCTSearch {
     UCTNode m_root{MOVE_NONE, 0.0f, 0.5f};
     std::atomic<int> m_nodes{0};
     std::atomic<int> m_playouts{0};
+    std::atomic<int> m_target_time{0};
```

Comment:
I think these need to be int64_t -- milliseconds since the epoch are greater than int sizes.

Reply:
Okay great, I'll swap that now